test/helpers/versions.spec.js

```diff
@@ -161,4 +240,19 @@ describe('helpers/versions', () => {
       versionsHelper.isPastLatest(qJson, '2.0.3').should.eql(true);
     });
   });
+  describe('.computeRangedVersion(currentVersion, newVersion)', () => {
```

Comment:
Agreed.

Though `computeRangeVersion` is not responsible for deciding which versions to upgrade to. It's only responsible for deciding what range character to place in front of a version.